src/pages/index.tsx

```diff
@@ -1,17 +1,72 @@
 import type { NextPage } from "next";
+import { Stack } from "@chakra-ui/react";
 import Head from "next/head";
 import site from "../../public/site.json";
 import Page from "../components/page/Page";
 
+import { PlannedMaintenance } from "../components/plannedMaintenance";
+import { OperationalStatus, Service } from "../components/OperationalStatus";
+import { PastIncidents } from "../components/pastIncidents";
+import { ServiceStatus } from "../components/OperationalStatus/OperationalStatus";
+
+import statusJsonData from "../../public/status.json";
+import incidentsJsonData from "../../public/incidents.json";
+import plannedMaintenanceJsonData from "../../public/planned_maintenance.json";
+
 const Home: NextPage = () => {
+  const statusData = statusJsonData;
```

Comment:
const [statusData, setStatusData] = useState<Insert Status Data Type Here>();

useEffect(() =>> {
// fetch the status data and then poll every 60 seconds for new data
}, [])

Reply:
Updated